src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/electric/machines/enchanting/AutoDisenchanter.java

```diff
@@ -152,10 +151,13 @@ private void transferEnchantments(ItemStack item, ItemStack book, Map<Enchantmen
     private boolean isDisenchantable(@Nullable ItemStack item) {
         if (item == null) {
             return false;
-        } else if (item.hasItemMeta() && item.getItemMeta().hasLore()) {
-            for (String lore : cantDisenchantLores.getValue()) {
-                if (item.getItemMeta().getLore().contains(ChatColors.color(lore))) {
-                    return false;
+        } else {
+            ItemMeta itemMeta = item.getItemMeta();
+            if (itemMeta != null && itemMeta.hasLore()) {
+                for (String lore : cantDisenchantLores.getValue()) {
+                    if (itemMeta.getLore().contains(ChatColors.color(lore))) {
```

Comment:
getLore() should be assigned to a variable to, it makes a copy each time its called

Reply:
I thought it was easier to read... Edited now.